src/main/java/com/github/ciomarabanu/aoc/_2021/Day10.java

```diff
@@ -0,0 +1,43 @@
+package com.github.ciomarabanu.aoc._2021;
+
+import com.github.ciomarabanu.aoc.utils.FileReaderFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Stack;
+
+public class Day10 {
+    public static void main(String[] args) {
+        var reader = FileReaderFactory.build();
+        var paranthesis = reader.readFileAsStrings("2021/day10");
+        System.out.println(getParenScore(paranthesis));
+    }
+
+    public static int getParenScore(List<String> paren) {
+        var points = new HashMap<Character, Integer>();
+        points.put(')', 3);
+        points.put(']', 57);
+        points.put('}', 1197);
+        points.put('>', 25137);
+
+        var score = 0;
+
+        for (var line : paren){
```

Comment:
Code is not formatted - missing empty space between `paren)` and `{`.

Reply:
Have you considered using a stream here and `flatMap`-ing it, with each lambda producing either an `Optional[Int]` (syntax error found) or an `Empty` (no syntax error found)?